tests/features/io_uring.t

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+. $(dirname $0)/../include.rc
+. $(dirname $0)/../volume.rc
+
+cleanup;
+
+TEST glusterd
+TEST pidof glusterd
+TEST $CLI volume create $V0 ${H0}:${B0}/${V0}-0
+TEST $CLI volume set $V0 performance.write-behind off
```

Comment:
Why do we disable write-behind?

Reply:
I wanted the writes to be wound immediately so that it appears in the brick profile info when we check it.